keyboards/playkbtw/helen80/rules.mk

```diff
@@ -7,6 +7,7 @@ BOOTLOADER = atmel-dfu
 # Build Options
 #   change yes to no to disable
 #
+VIA_ENABLE = yes
```

Comment:
This should go into a VIA keymap, not in the default keyboard config.

Reply:
oops, thanks for pointing that out, should have read more rules
changes are commited